plugins/repository-gcs/src/main/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobStore.java

```
@@ -411,4 +411,14 @@ private static String buildKey(String keyPath, String s) {
         assert s != null;
         return keyPath + s;
     }
+
+    @Override
+    public Map<String, Long> stats() {
+        try {
+            return storageService.stats(clientName).toMap();
+        } catch (IOException e) {
+            logger.error(() -> new ParameterizedMessage("unable to get GCS usage stats for client [{}]", clientName), e);
```

I'm not really sure if we should ignore the exception here or propagate it.